package.json

```diff
@@ -71,9 +69,14 @@
     "time-grunt": "1.2.1"
   },
   "scripts": {
-    "test": "grunt test; grunt lint",
-    "start": "node --harmony-arrow-functions index.js",
-    "startdev": "iojs --harmony-arrow-functions index.js --development -c"
+    "build": "babel --out-dir=dist index.js 'lib/**/*.js'",
+    "commit-dist": "git add dist/*",
+    "test": "grunt test",
+    "start": "babel-node index.js",
```

Comment:
take note.  this polyfills the node env. perf tests showed that in a simple case, it added ~0.8s. with many files, i'd imagine there is a 1.5s penalty. guessing!  look at the plus side. no more hybrid js.  if ur gonna write es6, write it fully! :)  could be RAD.